R/add_nc.R

```diff
@@ -0,0 +1,18 @@
+#'  Add nc
+#' 
+#' @description: This function takes a path as input and returns a modified path by appending "data.nc" to it.
```

Comment:
We should run this one by Roger later because I imagine the description here can be a lot more informative. However, myself I'm not sure what this function is for, so we can have those discussions later :)